docs/preview/03-Features/powershell/templates/arm.md

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
Added an issue that we do not forget to add a redirect to this new location: https://github.com/arcus-azure/arcus.scripting/issues/358